quantities.serialization/text.json/Extensions.cs

```diff
@@ -0,0 +1,12 @@
+using System.Text.Json;
+
+namespace Quantities.Serialization.Text.Json;
+
+public static class Extensions
+{
+    public static JsonSerializerOptions EnableQuantities(this JsonSerializerOptions options)
```

Comment:
The  respective `EnableQuantities` extension method for Newtonsoft.

Note: all other types are (at most) internal...